Source/WebCore/rendering/RenderFlexibleBox.h

```diff
@@ -231,7 +233,8 @@ class RenderFlexibleBox : public RenderBlock {
 
     mutable OrderIterator m_orderIterator { *this };
     int m_numberOfInFlowChildrenOnFirstLine { -1 };
-    
+    int m_numberOfInFlowChildrenOnLastLine { -1 };
```

Comment:
magic number, pls use optional instead.

Reply:
Changed both m_numberOfInFlowChildrenOnFirstLine and m_numberOfInFlowChildrenOnLastLine to std::optional<size_t>. I updated all of the reference also. Let me know if you see any issues with them.